lib/feedback.js

```diff
@@ -21,8 +21,13 @@ module.exports = function (input,options) {
     var texvcres = texvcjs.check(input,options);
     if (texvcres.status !== '+') {
         out.success = false;
-        out.status = texvcres.status;
-        out.detail = texvcres.details;
+        out.error = {
```

Comment:
It'd be better to have this set as `out.detail` because otherwise RESTBase will blacklist it and will not send it to the client.